package.json

```diff
@@ -70,6 +70,7 @@
     "intl": "^1.1.0",
     "jest": "^19.0.2",
     "lodash": "^3.10.1",
+    "marked": "^0.3.6",
```

Comment:
the package is abandoned. I would recommend using [alternative](https://www.npmjs.com/package/markdown-it)